cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocTool.java

```diff
@@ -105,10 +106,11 @@ public static void main(String[] args) throws Exception {
       conf = DataprocConf.create(properties);
     }
 
-    String imageVersion = commandLine.hasOption('i') ? commandLine.getOptionValue('i') : "1.2";
+    String imageVersion = commandLine.hasOption('i') ? commandLine.getOptionValue('i') : "1.3";
```

Comment:
Why don't we update this default to 2.0? 1.3 is currently unsupported by Dataproc.